--- src/gatt-client.c.old	2016-02-04 20:53:54.029797174 +0100
+++ src/gatt-client.c	2016-02-04 20:54:17.261795435 +0100
@@ -1205,9 +1205,6 @@
 	const char *sender = dbus_message_get_sender(msg);
 	struct notify_client *client;
 
-	if (!chrc->notifying)
-		return btd_error_failed(msg, "Not notifying");
-
 	client = queue_remove_if(chrc->notify_clients, match_notify_sender,
 							(void *) sender);
 	if (!client)
