*** src/Stack/Setup/Installed.hs.old	2017-06-05 11:03:36.226384136 +0200
--- src/Stack/Setup/Installed.hs	2017-06-05 11:03:54.257384713 +0200
*************** installDir :: (MonadReader env m, MonadT
*** 188,200 ****
             -> Tool
             -> m (Path Abs Dir)
  installDir programsDir tool = do
!     reldir <- parseRelDir $ toolString tool
!     return $ programsDir </> reldir
  
  tempInstallDir :: (MonadReader env m, MonadThrow m)
             => Path Abs Dir
             -> Tool
             -> m (Path Abs Dir)
  tempInstallDir programsDir tool = do
!     reldir <- parseRelDir $ toolString tool ++ ".temp"
!     return $ programsDir </> reldir
--- 188,200 ----
             -> Tool
             -> m (Path Abs Dir)
  installDir programsDir tool = do
!     reldir' <- parseRelDir $ toolString tool
!     return $ programsDir </> reldir'
  
  tempInstallDir :: (MonadReader env m, MonadThrow m)
             => Path Abs Dir
             -> Tool
             -> m (Path Abs Dir)
  tempInstallDir programsDir tool = do
!     reldir' <- parseRelDir $ toolString tool ++ ".temp"
!     return $ programsDir </> reldir'
